app/uk/gov/hmrc/tai/util/yourTaxFreeAmount/PersonalAllowance.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tai.util.yourTaxFreeAmount
+
+import uk.gov.hmrc.tai.model.domain.{PersonalAllowanceAgedPAA, PersonalAllowanceElderlyPAE, PersonalAllowancePA, TaxComponentType}
+
+object PersonalAllowance {
```

Comment:
Could this be a backend concern

Reply:
Hmmm, there may be areas of the code that use this API that need to distinguish between them. In this case we are just treating them as the same.

This could in theory go on the backend